frontend/public/components/filter-toolbar.tsx

```diff
@@ -445,7 +443,7 @@ export const FilterToolbar: React.FC<FilterToolbarProps> = ({
                   >
                     <div className="pf-v6-c-input-group co-filter-group">
                       {showSearchFiltersDropdown && (
-                        <DropdownInternal
+                        <ConsoleSelect
```

Comment:
The list page `ToolbarFilter` type filter and AutoCompleteInput are not working.

https://github.com/user-attachments/assets/acc207d7-5970-4e7c-88dd-2ae7ef7594bf